sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/TokenResult.java

```diff
@@ -212,6 +230,14 @@ public String status() {
         return this.status;
     }
 
+    /**
+     * Get account id.
+     * @return The account id value.
+     */
+    public String accountId() {
```

Comment:
```suggestion
    public String getAccountId() {
```

Reply:
will fix latter in another pr~

Reply:
Let's get that merged, then change everywhere when noone is modifying it.